client/ayon_nuke/addon.py

```diff
@@ -71,4 +72,4 @@ def get_launch_hook_paths(self, app):
         ]
 
     def get_workfile_extensions(self):
-        return [".nk"]
+        return [".nk"]
```

Comment:
Could you add the new line at end of file again to adhere to the code style of the repo?

Reply:
sorry didn't know what happened, I'll fix it right away